### PR TITLE
[plugin.video.catchuptvandmore] 0.2.27

### PR DESCRIPTION
**Pull request by sy6sy2**
### Description

[Version 0.2.27]

[CA - ICI Télé] Fix Catch-Up TV
[CA - TéléQuebec] Fix Catch-Up TV (New contents)
[BE - RTBF] Fix Live TV (Tipik)
[US - NY Times] Fix Website
[BE - Télésambre] Add Live TV
[BE - RTBF] Fix HTTP 400 during play catchup TV
[CA - Noovo] Fix Live TV and Rename V Télé to Noovo
[CH - RTS] Add DRM support for catchup TV
[WO - Arte] Fix some sub-categories not shown on Catchup TV
[FR - FranceTV] Add DRM support for catchup TV
[CA - TeleQuebec] Fix Live TV
[FR - MyCanal] Fix Catch-up TV
[FR - Gulli] Fix Live TV
[FR - BSmart] Fix Live TV
[FR - BFM Business] Fix Catch-up TV
[UK - DPLAY] Add Live TV and Catch-up TV (Quest, Quest Red, Really, ...)
[UK - QuestOD] Move code to Dplay
[CA - TV5Unis] Fix Catch-up TV

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.catchuptvandmore/resources/lib/channels/wo/tv5mondeplus.py [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+@Resolver.register
+def get_video_url(plugin,
+                  item_id,
+                  video_id,
+                  download_mode=False,
+                  **kwargs):
+
+    if get_kodi_version() < 18:
+        xbmcgui.Dialog().ok('Info', plugin.localize(30602))
```
Comment (enen92) [inline plugin.video.catchuptvandmore/resources/lib/channels/wo/tv5mondeplus.py]: Please localize info (here and on the other usages)

**Review by enen92 — CHANGES_REQUESTED** [review]
Minor fix and is good to go

**Comment by sy6sy2** [comment]
Should be ok now. Thank you @enen92!

**Review by enen92 — APPROVED** [review]

**Comment by sy6sy2** [comment]
Is something missing before the merge?

**Comment by enen92** [comment]
No sorry, Travis didn't run when I first approved and ended up forgetting about merging it :)

**Comment by sy6sy2** [comment]
Oh sorry, I didn't notice that. No problem 😉 
Thank you!

**Pull request merged**